### DISCUSSION TITLE
AppCenter doesn't open.

### DISCUSSION THREAD

**Discussion by GeneralAccess** [elementary/installer#734] [General]
When I click the AppCenter icon I get the rotating circle, but then nothing. I'd try starting it in a terminal to see what errors come up, but I don't know what the command is.
Edit: This is the same [as](https://github.com/elementary/appcenter/discussions/1930)

**Comment by danirabbit** [comment]
Closing as a duplicate of https://github.com/elementary/appcenter/discussions/1930
